CONTRIBUTING.md

```diff
@@ -17,3 +17,32 @@ license.
 By contributing documentation, examples, or any other non-code assets you agree
 to license your contribution under the CC BY 3.0 license. Attribution shall be
 given according to the current bylaws of this group.
+
+# Merge & Access Policy
+
+All Editors, Coordinators and Sponsors of specifications in draft stage have
```

Comment:
We remove access to the main repo after the draft/review stage but leave the groups intact for @mentions